allensdk/test/core/test_brain_observatory_cache.py

```diff
@@ -368,5 +375,46 @@ def test_find_specimen_reporter_line():
     cre = _find_specimen_reporter_line(s)
     assert cre is None
 
+from pkg_resources import resource_filename  # @UnresolvedImport
+import sys
+import json
+import logging
+
+if 'TEST_SESSION_ANALYSIS_REGRESSION_DATA' in os.environ:
+    data_file = os.environ['TEST_SESSION_ANALYSIS_REGRESSION_DATA']
+else:
+    data_file = resource_filename(__name__, '../brain_observatory/test_session_analysis_regression_data.json')
+
+@pytest.fixture(scope="module")
+def paths():
+    pyversion = sys.version_info[0]
+    logging.debug("loading " + data_file)
+    with open(data_file,'r') as f:
+        data = json.load(f)
+        return data[str(pyversion)]
+
+
+
+@pytest.mark.parametrize("session_type", ['a', 'b', 'c'])
```

Comment:
update this file:  '../brain_observatory/test_session_analysis_regression_data.json'

explicitly add the OEID as a key, e.g.:

  [ { 'python_version': 2,
       'ophys_experiment_id': '123457890',
       'analysis_file': '/path/to/analysis',
       'nwb': '/path/to/nwb'
     }]